src/metabase/models/collection.clj

```diff
@@ -49,15 +49,30 @@
   "The fixed ID of the trash collection."
   13371339)
```

Comment:
I kind of think we should change this, but I don't think it's been decided yet. I might keep it like this for now and, if we do want to change it, do it in a followup PR, since this is going into a feature branch. WDYT?